libnmstate/nm/active_connection.py

```diff
@@ -199,17 +200,16 @@ def remove_handlers(self):
         self.handlers = set()
         for handler_id in self.device_handlers:
             self.nmdevice.handler_disconnect(handler_id)
-        self.device_handlers = set()
 
 
 def _is_device_master_type(nmdev):
     if nmdev:
-        gobject = nmclient.GObject
+        gobject = GObject
```

Comment:
Is this needed? As `nmclient` is not needed anymore I think you can use `GObject` directly without the variable. Thanks!

Reply:
That save some typing on `from gi.repository import GObject`.

Reply:
Fixed.